src/main/java/step2/model/WinningLotto.java

```diff
@@ -0,0 +1,23 @@
+package step2.model;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class WinningLotto {
+
+	private List<Integer> winNumList;
+
+	public WinningLotto(List<Integer> winNumList) {
+		this.winNumList = winNumList;
+	}
+
+	public int countMatch(List<Integer> lotto) {
+		return winNumList.stream()
+			.filter(
+				target -> lotto.stream().distinct().anyMatch(Predicate.isEqual(target)))
+			.collect(Collectors.toList())
+			.size();
```

Comment:
다음과 같이 작성해볼 수도 있을 것 같아요.
```suggestion
		return (int) winNumList.stream()
			.filter(lotto::contains)
			.count();
```